### PR TITLE
Darren revised bio

### PR DESCRIPTION
**Pull request by tldlauser**
Bio is now revised.

### CODE REVIEW

File: content/who-we-are/our-team/contents.lr [hunk 1]
```diff
@@ -112,14 +112,14 @@ padding:
...
 
+Organizational health is a high priority. What does this have to do with me? Anyone who wants to know about me, will hear about what I have learned about team vitality (especially from Patrick Lencioni), because that is what I will be doing. Though I am quite happy to talk about organizational health, Pat is much better at teaching it, so I focus on living it.
+
 Healthy relationships are the highest earthly priority. This importance is primarily for the sake of those who are dearest, but healthy relationships are also the most crucial ingredient for both physical health and organizational health.
```
[the remark this answers is not in the document]
Comment (tldlauser) [reply content/who-we-are/our-team/contents.lr:123]: Cool, makes sense. I added a wee bit to summarize what I mean.

File: content/who-we-are/our-team/contents.lr [hunk 2]
```diff
@@ -112,14 +112,14 @@ padding:
...
 
 Physical health is a high priority. Some people joke about exercising and healthy eating, saying they will just be in the nursing home longer. I get the joke, but don't take it seriously. Hopefully you are aware that it is backwards. Yes, the statistics show that making healthy choices does add days to your life, but it also adds life to your days. Of course, I realize there are many factors that could cause someone to need elderly care, but if I ever end up in a nursing home, I don't want it to be caused by my poor choices. Less illness, less severity of illness, less time to recover from illness and injury, more clarity of thought, more energy, more power, more vitality -- those are some of the benefits of a healthy lifestyle!
 
+Organizational health is a high priority. What does this have to do with me? Anyone who wants to know about me, will hear about what I have learned about team vitality (especially from Patrick Lencioni), because that is what I will be doing. Though I am quite happy to talk about organizational health, Pat is much better at teaching it, so I focus on living it.
```
[the remark this answers is not in the document]
Comment (tldlauser) [reply content/who-we-are/our-team/contents.lr:121]: Yes, Ben was very helpful, both with this and all of the content.

File: content/who-we-are/our-team/contents.lr [hunk 3]
```diff
@@ -112,14 +112,14 @@ padding:
...
+After living in Ireland for 13 years, some people say we moved back to America. But, really, we did not move back, we moved forward -- to Oklahoma! We truly enjoyed our Ireland adventure. Now we are ready for our Oklahoma adventure.
 
 Physical health is a high priority. Some people joke about exercising and healthy eating, saying they will just be in the nursing home longer. I get the joke, but don't take it seriously. Hopefully you are aware that it is backwards. Yes, the statistics show that making healthy choices does add days to your life, but it also adds life to your days. Of course, I realize there are many factors that could cause someone to need elderly care, but if I ever end up in a nursing home, I don't want it to be caused by my poor choices. Less illness, less severity of illness, less time to recover from illness and injury, more clarity of thought, more energy, more power, more vitality -- those are some of the benefits of a healthy lifestyle!
```
[the remark this answers is not in the document]
Comment (tldlauser) [reply content/who-we-are/our-team/contents.lr:119]: As we discussed, I feel the need to assure people I am not blaming them for being in a nursing home. I did shorten this some anyway.

**Review by BenjaminEchelmeier — APPROVED** [review]
@alan-cugler, @nixjdm I think this is pretty well ready--can possibly merge it as soon as Sam's is ready too.

**Comment by alan-cugler** [comment]
I think its good!

**Pull request merged**